test/fixtures/generic/encounter.json

```diff
@@ -8,6 +8,11 @@
         "Annual_Physical": {
             "type": "Encounter",
             "wellness": true,
+            "codes": [{
```

Comment:
Yes, in providers_test.rb test_utilization_encounter, I call perform_encounter in states.rb.  All the other tests I looked at call perform_encounter with record_encounter = false, but I want to test the increment_encounters, which requires record_encounter to be true.  Somewhere during the record process, it throws an error because codes is null.  Is there a better way to test this without adding codes to the wellness encounter?